backend/routes/giphy.js

```diff
@@ -22,13 +20,17 @@ router.route("/").get(async (req, res) => {
 router.route("/search").get(async (req, res) => {
   try {
     const { searchTerm, limit, offset } = req.query;
-    const { data } = await getSearchedImages(apiSources.giphy.id, limit, searchTerm, offset);
+    const { data } = await getSearchedImages(
```

Comment:
this change is not related to your story. Please just push what is necessary to fix the issue. It makes harder/longer for reviewers to read.